### ISSUE TITLE
Geocoder requires an API key to use options parameters even if global key is set

### ISSUE THREAD

**Issue by rmglennon** [mapzen/mapzen.js#374]
I set a mapzen.js global apiKey, and was finding that the geocoder still requires its own API key to be passed for the geocoder options to work.

Here's some code to reproduce: 

```
<script>
      L.Mapzen.apiKey = "your-mapzen-api-key";
      var map = L.Mapzen.map("map", {
        center: [47.61033,-122.31801],
        zoom: 16,
      });
      var geocoderOptions = {
        autocomplete: false
      };
      var geocoder = L.Mapzen.geocoder(geocoderOptions);
      geocoder.addTo(map);
    </script>
```

but could not get the geocoder to use my options. In this case, it still continued to use autocomplete in the search box when it should have switched to the search endpoint.

It did work with `var geocoder = L.Mapzen.geocoder("", geocoderOptions);` where I passed an empty set (or an actual key) for the geocoder API key.

**Comment by hanbyul-here** [comment]
This fix was just released (`v0.6.9`)

**Comment by rmglennon** [comment]
thanks for the fix and status note, @hanbyul-here!
